wordvector_classify.py

```diff
@@ -0,0 +1,89 @@
+from libs.dataset import DataSet
+from libs.gen_wordvectors import WordVector
+from libs.generate_test_splits import generate_hold_out_split, kfold_split, get_stances_for_folds
+from libs.score import score_submission
+
+class WordVectorClassify:
+
+    def __init__(self):
+        self.dataset = DataSet()
+
+    def do_validation(self):
+        folds, hold_out = kfold_split(self.dataset, n_folds=10)
+        fold_stances, hold_out_stances = get_stances_for_folds(self.dataset, folds, hold_out)
+
+        labeled_feat_dict = {}
+
+        print "Generating features for each fold"
+        for fold_id in fold_stances:
+            print "Generating features for fold ", fold_id
+            bodies = folds[fold_id]
+            stances = fold_stances[fold_id]
+
+            wordvectors = WordVector().gen_wordvectors(
+                    self.dataset, bodies, stances)
+            labeled_feature_set = []
+            for i in range(len(stances)):
+                labeled_feature = ({
+                    'word_vectors':wordvectors[i]},
+                    self._process_stance(stances[i]['Stance']))
+                labeled_feature_set.append(labeled_feature)
+
+            labeled_feat_dict[fold_id] = labeled_feature_set
+
```

Comment:
I have yet to properly test anything below this line. Working on it. The above for loop takes 15 minutes.